projects/apollox/index.js

```diff
@@ -15,14 +19,14 @@ const TOKEN_BUSD = "0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56";
 const TOKEN_CAKE = "0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82";
 const TOKEN_BABY = "0x53e562b9b7e5e94b81f10e96ee70ad06df3d2657";
 const TOKEN_LEOS = "0x2c8368f8f474ed9af49b87eac77061beb986c2f1";
-const TOKEN_USDC = "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d";
+const TOKEN_USDC = "0x8ac76a51cc950d982268b83fe1ad97b32cd580d";
```

Comment:
think these fixes are incorrect, USDC token points to nothing and TOKEN_MDX is invalid: https://www.bscscan.com/search?f=0&q=0x8ac76a51cc950d982268b83fe1ad97b32cd580d